### DISCUSSION TITLE
Getting started with Oracle NoSQL - from high-availability to extreme-availabilty

### DISCUSSION THREAD

**Discussion by dario-vega** [oracle/nosql#46] [Announcements]
The code in this repository was used in **Getting started with an Oracle NoSQL workshops**  showcased in NoSQL events around the world.

https://github.com/oracle/nosql-examples/tree/master/examples-nosql-cluster-deployment

This repository provides you guidelines and examples allowing you to **learn how to deploy a NoSQL cluster/NoSQL store** (also referred to as the topology). We will learn how to deploy the most popular topologies:
- single node
- multi node in only one zone
- multi node in multiple primary zones without affinity
- multi node in multiple primary zones with affinity
- multi node in multiple zones (2 primary) + arbiters
- multi node in multiple zones (2 primary/1 secondary) + arbiters
- multi node in multiple zones (1 primary/2 secondary) + arbiters
- multi region deployment

And how to modify an existing configuration
- Increase availability - modify the replication factor
- Elastic Expansion and Rebalancing - scale up/down - scale in/out

There are 2 ways to configure a NoSQL cluster, one using a **secure** configuration and another one using a **non-secure** configuration. The primary difference is in the way access is performed to NoSQL store. We recommend using the secure setup, albeit additional steps are needed during set up.

**Comment by mikebrey** [comment]
reviewing the content on this page.
